packages/component/src/SendBox/SendButton.js

```diff
@@ -8,22 +8,29 @@ import SendIcon from './Assets/SendIcon';
 const connectSendButton = (...selectors) => connectToWebChat(
   ({
     language,
+    sendBoxValue,
+    setSendBox,
     submitSendBox
   }) => ({
-    click: submitSendBox,
+    click: () => {
+      setSendBox(sendBoxValue.trim());
+      submitSendBox();
+    },
     language
   }),
   ...selectors
 )
 
 export default connectSendButton()(
   ({ click, language }) =>
-    <IconButton
-      alt={ localize('Send', language) }
-      onClick={ click }
-    >
-      <SendIcon />
-    </IconButton>
+    <div>
```

Comment:
jw, can this be a `<React.Fragment>` or does it need to be a `<div>`?

Reply:
Not sure. I used a div since the attachment button was wrapped in one.